### ISSUE TITLE
Confluence Style and Assignee/Notebook List: assignee is ignored if there is a date

### ISSUE THREAD

**Issue by taw00** [CalebJohn/joplin-inline-todo#25]
In my notebook "00 SCRATCH and UNFILED" I edited my "!!!SCRATCH" note …

```markdown
- [ ] Testing confluence format. +Joplin Testing things.
- [ ] @jim //2023-10-19 What the heck? +someLabel
- [ ] Testing a 3rd todo. @todd
```

Produces a "Todo Summary" of …

```markdown
# DUE
- [!!!SCRATCH](:/6a95fc8e8ff44368adf3ebd422cd6878): 2023-10-19 What the heck? +someLabel

## 00 SCRATCH and UNFILED
- [!!!SCRATCH](:/6a95fc8e8ff44368adf3ebd422cd6878): Testing confluence format. Testing things. +Joplin

# TODD
## 00 SCRATCH and UNFILED
- [!!!SCRATCH](:/6a95fc8e8ff44368adf3ebd422cd6878): Testing a 3rd todo.
```

Shouldn't `@jim` be acknowledged? Or have that item be in it's own named category?

Thanks. -t

**Comment by CalebJohn** [comment]
> Shouldn't @jim be acknowledged?

Yes it should. I'll add it next to the tags.

> Or have that item be in it's own named category?

It's an intentional choice to place the items with a due date under a separate heading at the top. There fear is that with many todo items, they'll have a tendency to get lost.

**Comment by CalebJohn** [comment]
This will be fixed in v1.7.0

**Comment by taw00** [comment]
> It's an intentional choice to place the items with a due date under a separate heading at the top. There fear is that with many todo items, they'll have a tendency to get lost.

Makes sense.
